drivers/st25r3911b/Kconfig

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig ST25R3911B_LIB
+	bool "NFC ST25R3911B library"
+	help
+	  Enable the NFC ST25R3911B library
+
+if ST25R3911B_LIB
+
+config ST25R3911B_SPI_FREQ
+	int "ST25R3911B SPI frequency"
+	default 4000000
+	help
+	  ST25R3911B SPI frequency
+
+config ST25R3911B_SPI_PORT
+	int "ST25R3911 SPI port"
+	default 0
+	help
+	  Configure a SPI port for ST25R3911B
+
+config ST25R3911B_IRQ_PIN
+	int "ST25R3911B IRQ pin"
+	default 3
```

Comment:
Why `3`?
I'm not convinced that any default value should be provided here (same applies to `ST25R3911B_SPI_PORT`). User should be aware of what is being configured.

Reply:
Because, `3` is the number of GPIO pin where shield has physically connected IRQ signal. The configuration is left in cases if someone would like to connect it to other places for tipping using cables